fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -87,8 +87,8 @@ def self.upload_dsym(params, path)
         UI.message("Uploading '#{path}'...")
         command = []
         command << File.expand_path(params[:binary_path]).shellescape
-        command << "-a #{params[:api_token]}" if params[:api_token]
         command << "-gsp #{params[:gsp_path].shellescape}" if params[:gsp_path]
+        command << "-a #{params[:api_token]}" if params[:api_token] && !params[:gsp_path]
```

Comment:
Can we actually make this an `if/else`? I think that would make this logic a bit clearer 🙃 Something like...
```rb
if params[:gsp_path]
  command << "-gsp #{params[:gsp_path].shellescape}"
elsif params[:api_token]
  command << "-a #{params[:api_token]}"
end
```

Reply:
Made the change.  Sorry i didn't check this PR for so long.